lint/style.py

```diff
@@ -46,16 +43,6 @@ def load_gutter_icons():
         colorize = True
 
     new_gutter_dict["colorize"] = colorize
-    dir_path, _ = os.path.split(theme_file)
-    pck_path = sublime.packages_path().split("/Packages")[0]
-    abs_dir = os.path.join(pck_path, dir_path)
-
-    png_files = glob(os.path.join(abs_dir, "*.png"))
```

Comment:
Obviously, we can't `glob` through a zip file.

Reply:
Indeed, we can't pre-emptively loop over the file system to find the available icons for a given theme.